src/clj/game/cards/programs.clj

```diff
@@ -31,6 +31,17 @@
    {:events {:jack-out {:effect (effect (gain-credits 1))
                         :msg "gain 1 [Credits]"}}}
 
+   "Bankroll"
+   {:implementation "Bankroll gains credits automatically."
+    :events {:successful-run {:effect (effect (add-counter card :credit 1)
+                                              (system-msg "places 1 [Credit] on Bankroll."))}}
+    :abilities [{:label "[Trash]: Take all credits from Bankroll."
+                 :async true
+                 :effect (req (let [credits-on-bankroll (get-counters card :credit)]
+                                (wait-for (trash state :runner card {:cause :ability-cost})
+                                          (gain-credits state :runner credits-on-bankroll)
+                                          (system-msg state :runner "trashes Bankroll and takes all credits from it."))))}]}
```

Comment:
Can this message give the # of credits instead?

Reply:
Keep messages as "takes # credits from it" or change it to "takes # [Credits] from it"?